app/src/androidTest/java/com/nervousfish/nervousfish/test/LoginActivitySteps.java

```diff
@@ -35,39 +34,24 @@ public LoginActivitySteps(LoginActivity activityClass) {
     }
 
     @Given("^I have a LoginActivity")
-    public void I_have_a_LoginActivity() {
+    public void i_have_a_LoginActivity() {
         assertNotNull(getActivity());
     }
 
-    @When("^I input email (\\S+)$")
-    public void I_input_email(final String email) {
-        onView(withId(R.id.email)).perform(typeText(email));
-    }
-
     @When("^I input password \"(.*?)\"$")
-    public void I_input_password(final String password) {
+    public void i_input_password(final String password) {
         onView(withId(R.id.password)).perform(typeText(password));
     }
 
     @When("^I press submit button$")
-    public void I_press_submit_button() {
+    public void iPressSubmit() {
         onView(withId(R.id.submit)).perform(scrollTo()).perform(click());
     }
 
-    @Then("^I should see error on the (\\S+)$")
-    public void I_should_see_error_on_the_editTextView(final String viewName) {
-        int viewId = (viewName.equals("email")) ? R.id.email : R.id.password;
-        int messageId = (viewName.equals("email")) ? R.string.msg_email_error : R.string.msg_password_error;
-
-        onView(withId(viewId)).check(matches(hasErrorText(getActivity().getString(messageId))));
-    }
-
     @Then("^I should (true|false) auth error$")
     public void I_should_see_auth_error(boolean shouldSeeError) {
         if (shouldSeeError) {
             onView(withId(R.id.error)).check(matches(isDisplayed()));
-        } else {
```

Comment:
Why not keep the else branch? As long as the description for the test method is `^I should (true|false) auth error$` it should be able to handle both cases...